lib/Controller/Configuration/ResponseHeader.php

```diff
@@ -28,27 +26,29 @@
  *
  * @Annotation
  */
-class ResponseHeader extends ConfigurationAnnotation
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
+final class ResponseHeader
```

Comment:
This needs deprecation notice on 10.6 to switch from `annotation` to `attribute`.

Reply:
please see https://github.com/pimcore/pimcore/pull/14203